components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -188,7 +157,10 @@ func TheiaRootHandler(infoProvider WorkspaceInfoProvider) RouteHandler {
 			reslv = staticTheiaResolver
 		}
 		resolver := func(config *Config, req *http.Request) (*url.URL, error) {
-			if ideQueryMatch(req, nil) {
+			if isWebsocketRequest(req) {
+				return staticTheiaResolver(config, req)
```

Comment:
this doesn't make sense to me.
1. The static Theia server can never answer websocket requests properly (it's just a static web server), and
2. if that were desired behaviour, we'd be breaking blobserve support.

What was the intention of exception?

Reply:
it is just me trying to figure out where to make sure that websocket requests don't hit blobserve, but it does not seem to necessary now at all.

Reply:
I removed it completely.